### PR TITLE
TCOMP-595:add missing libs to S3 on windows

### PR DESCRIPTION
**Pull request by gonchar-ivan**
**What is the current behavior?** (You can also link to an open issue here)

tcomp-service on Windows fails when S3 component is called

**What is the new behavior?**

tcomp-service contains hadoop winutils, which is required by S3 component

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://jira.talendforge.org/browse/TCOMP-595

### CODE REVIEW

**Comment by gonchar-ivan** [comment]
This PR creates one more output for components-maven-repo module. It is artifact with "binaries" classifier. It is assumed to store all binaries in this .zip artifact. 
Another solution is to unpack winutils just in tcomp-service all components. But as components-maven-repo is a point of component configuration I decided to put binaries there.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/730/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/730/)

**Comment by sgandon** [comment]
@gonchar-ivan could you also activate all S3 integration tests too in this PR, similarly to the commit https://github.com/Talend/components/commit/86cf1a35895329fd470cf3d0510c234d68db3390

File: components/components-fileio/simplefileio-runtime/src/test/java/org/talend/components/simplefileio/runtime/s3/S3DatasetRuntimeTestIT.java [hunk 1]
```diff
@@ -49,7 +47,6 @@ public void reset() {
     }
 
     @Test
-    @Ignore("It's a very slowly test, need 10 more mins")
```
Comment (gonchar-ivan) [inline components/components-fileio/simplefileio-runtime/src/test/java/org/talend/components/simplefileio/runtime/s3/S3DatasetRuntimeTestIT.java]: Not sure whether it is worth to remove ignore on this test

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/786/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/786/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/787/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/787/)

**Comment by sgandon** [comment]
nice, thanks a lot.

**Pull request merged**